### PR TITLE
ML-442: empty constructor binding

### PR DESCRIPTION
**Pull request by iryna-melnyk**
A tiny change which helps to reduce a number of lines for tensor initialisation in etch:
var x = Tensor();
x.fromString(string);

### CODE REVIEW

File: libs/vm-modules/src/math/tensor/tensor.cpp [hunk 1]
```diff
@@ -85,6 +90,7 @@ void VMTensor::Bind(Module &module, bool const enable_experimental)
 
   auto interface =
       module.CreateClassType<VMTensor>("Tensor")
+          .CreateConstructor(&VMTensor::EmptyConstructor)
```
Comment (baykaner) [inline libs/vm-modules/src/math/tensor/tensor.cpp]: you still need to charge for doing that!

Also you should make this an experimental binding

File: libs/vm-modules/tests/unit/ml/ml_model_tests.cpp [hunk 2]
```diff
@@ -1096,9 +1093,7 @@ TEST_F(VMModelTests, model_sequential_flatten_tensor_data)
 {
```
Comment (baykaner) [inline libs/vm-modules/tests/unit/ml/ml_model_tests.cpp:1093]: add some tests for empty tensor construction and interaction (test how it fails when you assign values to it, check you can reshape it and then assign values etc.)

**Review by baykaner — CHANGES_REQUESTED** [review]

File: libs/vm-modules/src/math/tensor/tensor.cpp (hunk 1 above)
Comment (iryna-melnyk) [reply libs/vm-modules/src/math/tensor/tensor.cpp]: ok, got it

File: libs/vm-modules/tests/unit/ml/ml_model_tests.cpp (hunk 2 above)
Comment (iryna-melnyk) [reply libs/vm-modules/tests/unit/ml/ml_model_tests.cpp:1093]: Done

**Review by baykaner — APPROVED** [review]
